.github/workflows/reusable-database-deploy.yml

```diff
@@ -44,6 +49,19 @@ jobs:
           op: decode
           in: ${{ inputs.azResourceGrpNetworkEncrypted }}
 
+      - name: Check if E2E CosmosDb exists ##Check if E2E DB exists determining if the DB needs data seeded downstream
```

Comment:
re-running the seed on an already seeded database risks corrupting existing data. So we check this to conditionally run the seed operation downstream